src/instructlab/train/linux_train.py

```diff
@@ -297,7 +297,7 @@ def model_generate(user, **kwargs):
             input_ids=input_ids,
             max_new_tokens=256,
             pad_token_id=tokenizer.eos_token_id,
-            temperature=0.7,
+            temperature=1.0,
```

Comment:
Why this change?

Reply:
Spoke with @alimaredia, this change is not needed here